tests/test_baker.py

```diff
@@ -633,6 +637,130 @@ def test_allows_save_kwargs_on_baker_make(self):
         assert owner == dog1.owner
         assert owner == dog2.owner
 
+    def test_allow_user_to_specify_database_via_save_kwargs_for_retro_compatibility(
+        self,
+    ):
+        profile = baker.make(models.Profile, _save_kwargs={"using": settings.EXTRA_DB})
+        qs = models.Profile.objects.using(settings.EXTRA_DB).all()
+
+        assert 1 == qs.count()
```

Comment:
```suggestion
        assert qs.count() == 1
```
In my opinion, having the result on the left and the expected result on the right is more intuitive but this is a matter of style.

Reply:
Yep, for sure! Fixed in 1d1855886bfd5bc3da7a8c3655a2e3b7a458aa66